src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/ReflectionRuleRegistration.cs

```diff
@@ -1,14 +1,15 @@
 ﻿using System.Collections.Generic;
+using NBitcoin.Rules;
 using Stratis.Bitcoin.Consensus.Rules;
 using Stratis.Bitcoin.Features.SmartContracts.ReflectionExecutor.Consensus.Rules;
 
 namespace Stratis.Bitcoin.Features.SmartContracts.ReflectionExecutor
 {
     public sealed class ReflectionRuleRegistration : IRuleRegistration
     {
-        public IEnumerable<ConsensusRule> GetRules()
+        public List<IConsensusRule> GetRules()
```

Comment:
Is it possible to revert to IEnumerable, since I guess the return list will be only read?

Reply:
OK I will try and change to ICollection 👍